src/utils/isOnlyEmoji.ts

```diff
@@ -1,8 +1,35 @@
 const emojiRegex =
-  /(<a?(:\w+:\d+)>|\p{Emoji_Presentation}|\p{Extended_Pictographic}|\p{Emoji_Component})\s*/gu
+  /(<a?(:\w+:\d+)>|\p{Emoji_Presentation}|\p{Extended_Pictographic}|\p{Emoji_Component}|:\w+:\s*)/gu
 
 export default (msg: string): boolean => {
   const emoji = msg.match(emojiRegex)
+  //the issue of #99(https://github.com/creatorsgarten/kaogeek-discord-bot/pull/99) only happend with only number message,
+  //so detect number in message and return false if there is.
+  if (emoji !== null) {
+    const unicoded = emoji.map((emo) => {
+      return emo.codePointAt(0)?.toString(16)
+    })
+    for (const i of unicoded) {
+      if (i !== undefined && isNumber(i)) {
+        return false
+      }
+    }
+  }
+  return emoji !== null && emoji.join('').trim() === msg.replace(/\s/g, '')
+}
 
-  return emoji !== null && emoji.join('').trim() === msg.trim()
+function isNumber(input: string): boolean {
+  const numberUnicode = [
+    '30',
+    '31',
+    '32',
+    '33',
+    '34',
+    '35',
+    '36',
+    '37',
+    '38',
+    '39',
+  ]
+  return numberUnicode.includes(input)
```

Comment:
I think remove `.toString(16)`
and use
```ts
function isNumber(input: number): boolean {
  return input >= 0x30 && input <= 0x39
```
would works?

Reply:
I will be back to these discuss https://github.com/kaogeek/kaogeek-discord-bot/pull/109#discussion_r1200684597

Reply:
it is same to these discuss https://github.com/kaogeek/kaogeek-discord-bot/pull/109#discussion_r1201978058 ??

Reply:
Yeah, I'm just skip `.toString(16)`.
Also comparison with string in js is weird.